pkgs/development/python-modules/eventlet/default.nix

```diff
@@ -29,9 +28,8 @@ buildPythonPackage rec {
   propagatedBuildInputs = [
     dnspython
     greenlet
-    pyopenssl
     six
-  ] ++ lib.optional (pythonOlder "3.5") [
+  ] ++ lib.optionals (pythonOlder "3.5") [
     monotonic
   ];
```

Comment:
The oldest supported python is 3.7 so this can be deleted.

Reply:
According to their setup.py not https://github.com/eventlet/eventlet/blob/master/setup.py#L44